Dockerfile

```diff
@@ -5,8 +5,12 @@
 # Build container
 FROM golang:1.14 AS builder
 
+# overwrite GOARCH with --build-arg GOARCH=arm64 when invoking docker build
+ARG GOARCH="amd64"
+
 WORKDIR /src
 COPY . .
+RUN export $ARCH
```

Comment:
I don't think this explicit export is required, at least when I tested it. (I don't have a raspberry pi handy to test on, the container appeared to be arm64.)